backup.yml

```diff
@@ -0,0 +1,19 @@
+# file: backup.yml
+
+- name: Backup OpenNMS
+  hosts: core
+  user: root
+  become: true
+  roles:
+    - opennms_backup
+  vars:
+   timestamp: "{{ ansible_date_time.date }}_{{ ansible_date_time.hour }}{{ ansible_date_time.minute }}"
+
+# - name: Backup PostgreSQL
```

Comment:
commented because the PG dump is not working yet. But you can see the structure I am following here.